### PR TITLE
Add skill icons and key-bindings to all builds

### PR DESCRIPTION
**Pull request by MuwuM**
### Find out how to use skill icons in your build [here](https://github.com/accessibilitywars/aw2-builds/wiki/Making-builds#how-to-use-skill-key-bindings-in-a-build).

Currently the following keyboard layouts are supported:
```
        US (default) [QWERTY]
        UK (British) [QWERTY]
        German [QWERTZ]
        Switzerland [QWERTZ]
        French [AZERTY]
        Spanish (Spain) [QWERTY]
        Spanish (Latin America) [QWERTY]
        Arabic (Egypt/101) [QWERTY]
```
  You can read [here](https://github.com/accessibilitywars/aw2-builds/wiki/Making-keyboard-layout-files) how to add more.

This PR also adds minor improvements to the build rotations like highlighting in which tool-kit to start engi-rotations.

This PR solves #1

### CODE REVIEW

**Comment by MuwuM** [comment]
There is still anissue with [Condi Firebrand](https://accessibilitywars.com/condi-firebrand/)

**there is** 
> You are expected to use Litany of Wrath, your heal, when its available.

But the build does not use `Litany of Wrath` but the heal sigil.

Should I 
1. remove that sentence 
**OR**
2. change the heal to `Litany of Wrath`

**Comment by MuwuM** [comment]
This PR can be previewed at https://muwum.github.io/ until it is merged.

**Comment by berdandy** [comment]
> There is still anissue with [Condi Firebrand](https://accessibilitywars.com/condi-firebrand/)
> 
> **there is**
> 
> > You are expected to use Litany of Wrath, your heal, when its available.
> 
> But the build does not use `Litany of Wrath` but the heal sigil.
> 
> Should I
> 
>     1. remove that sentence
>        **OR**
> 
>     2. change the heal to `Litany of Wrath`

This appears to be an error in the source (I have confirmed it is in the original spreadsheet). For now, let's go with option 1, but I'll reach out to jupiter to confirm just in case it was intended to be Litany.

**Comment by MuwuM** [comment]
Please review and merge, once you have the time for it 😄

**Comment by MuwuM** [comment]
Don't merge, yet. I'm changing it to use @discretize/gw2-ui-new

**Comment by MuwuM** [comment]
The changes should be done. Preview is available at https://muwum.github.io/ until merged

File: docs/src/entry.jsx
Comment (berdandy) [inline docs/src/entry.jsx]: This is not a fallback. This is an override. If Anet does update the API to include the skill, we'll still use our fallback data. Either change the logic to reflect that, or the name to make it clear. I'd prefer an actual fallback, but name change is relatively easier.

**Review by berdandy — CHANGES_REQUESTED** [review]
VERY surface-level review, there's a LOT there. Only one little nagging thing I noticed that might trip up future-us.

File: docs/src/entry.jsx
Comment (MuwuM) [reply docs/src/entry.jsx]: I'm now providing a real fallback. For a better solution we should add that functionality to gw2-ui

Comment (MuwuM) [reply docs/src/entry.jsx]: There is a solution in discussion already in discretize discord, so once a proper solution exits, I can remove my fallback-wrapper code. The planned solution will work fine with our custom wrapper

Comment (MuwuM) [reply docs/src/entry.jsx]: Solution is there: https://github.com/discretize/discretize-ui/blob/main/gw2-ui/src/gw2api/overrides/data/skills.ts

**Review by berdandy — APPROVED** [review]
LGTM 👍

**Pull request merged**
